packages/expo-sqlite/ios/SQLiteModuleNext.swift

```diff
@@ -38,52 +38,54 @@ public final class SQLiteModuleNext: Module {
       }
     }
 
-    AsyncFunction("deleteDatabaseAsync") { (databaseName: String) in
-      try deleteDatabase(databaseName: databaseName)
+    AsyncFunction("deleteDatabaseAsync") { (databasePath: String) in
+      try deleteDatabase(databasePath: databasePath)
     }
-    Function("deleteDatabaseSync") { (databaseName: String) in
-      try deleteDatabase(databaseName: databaseName)
+    Function("deleteDatabaseSync") { (databasePath: String) in
+      try deleteDatabase(databasePath: databasePath)
     }
 
-    AsyncFunction("importAssetDatabaseAsync") { (databaseName: String, assetDatabasePath: String, forceOverwrite: Bool) in
-      guard let path = pathForDatabaseName(name: databaseName) else {
-        throw Exceptions.FileSystemModuleNotFound()
-      }
+    AsyncFunction("importAssetDatabaseAsync") { (databasePath: String, assetDatabasePath: String, forceOverwrite: Bool) in
+      let path = try ensureDatabasePathExists(path: databasePath)
       let fileManager = FileManager.default
-      if fileManager.fileExists(atPath: path.absoluteString) && !forceOverwrite {
+      if fileManager.fileExists(atPath: path.standardizedFileURL.path) && !forceOverwrite {
```

Comment:
Here and below `absoluteString` is replaced with `standardizedFileUrl.path` because otherwise, it fails if the path points to the shared container